MMR.Randomizer/RandomizationException.cs

```diff
@@ -8,4 +8,11 @@ public RandomizationException(string message) : base(message)
         {
         }
     }
+    public class ROMOverflow : Exception
```

Comment:
Should rename it to ROMOverflowException and move to its own file.

Reply:
so changed